modules/@apostrophecms/area/ui/apos/components/AposAreaWidget.vue

```diff
@@ -41,13 +41,13 @@
           <li class="apos-area-widget__breadcrumb" data-apos-widget-breadcrumb="0">
             <AposButton
               type="quiet"
-              @click="foreign ? $emit('edit', i) : false"
+              @click="foreign ? $emit('edit', i) : null"
+              @dblclick.native="(!foreign && !isContextual) ? $emit('edit', i) : null"
               :label="foreign ? {
                 key: 'apostrophe:editWidgetType',
                 label: $t(widgetLabel)
               } : widgetLabel"
               tooltip="apostrophe:editWidgetForeignTooltip"
```

Comment:
We could remove this tooltip when contextual since we can't edit from de modal:
```javascript
:tooltip="!isContextual && 'apostrophe:editWidgetForeignTooltip'"
```